packages/twenty-server/src/workspace/messaging/producers/messaging-producer.ts

```diff
@@ -0,0 +1,26 @@
+import { Inject, Injectable } from '@nestjs/common';
+
+import { MessageQueues } from 'src/integrations/message-queue/message-queue.constants';
+import { MessageQueueService } from 'src/integrations/message-queue/services/message-queue.service';
+import {
+  FetchMessagesJob,
+  FetchMessagesJobData,
+} from 'src/workspace/messaging/jobs/fetch-messages.job';
+
+@Injectable()
+export class MessagingProducer {
+  constructor(
+    @Inject(MessageQueues.messagingQueue)
```

Comment:
@Weiko As discuss, I think it's better to be based on `@nestjs/bull` api and create a decorator `@InjectQueue` that is going to strongly type it